### PR TITLE
Feature/k  1295 add endpoints to getting and update availability restrictions for travel click channel

### PR DESCRIPTION
**Pull request by AlexeyShok**
Added error codes to komoro contracts. Fixed get request

### CODE REVIEW

**Review by DaniyarAskarov — APPROVED** [review]

File: HappyTravel.Komoro.Api/Services/Statics/RoomTypeService.cs [hunk 1]
```diff
@@ -30,6 +30,10 @@ public async Task<int> GetId(string roomTypeCode)
 
         return property?.Id ?? 0;
     }
+    
+    
+    public async Task<bool> IfExist(string roomTypeCode)
```
Comment (hasmikharutyunyan) [inline HappyTravel.Komoro.Api/Services/Statics/RoomTypeService.cs:35]: `IsExist` ?

File: HappyTravel.Komoro.TravelClickChannelManager/Infrastructure/ErrorHelper.cs [hunk 2]
```diff
@@ -6,14 +6,16 @@ namespace HappyTravel.Komoro.TravelClickChannelManager.Infrastructure;
 
 public static class ErrorHelper
 {
-    public static Error GetError(ErrorWarningTypes errorWarningType, ErrorCodes errorCode)
+    public static Error GetError(ErrorWarningTypes errorWarningType, ErrorCodes errorCode, string? ObjectCode)
```
Comment (hasmikharutyunyan) [inline HappyTravel.Komoro.TravelClickChannelManager/Infrastructure/ErrorHelper.cs:9]: I think `ObectCode` name is not understandable

**Review by oleg-n-k — APPROVED** [review]

File: HappyTravel.Komoro.Api/Services/Availabilities/AvailabilityRestrictionService.cs [hunk 3]
```diff
@@ -21,29 +23,74 @@ public AvailabilityRestrictionService(KomoroContext komoroContext, IDateTimeOffs
...
             .ToListAsync();
+
+        return (availabilityRestrictions, errorDetailsList);
     }
 
 
-    public async Task Update(int supplierId, List<AvailabilityRestriction> availabilityRestrictions)
+    public async Task<List<ErrorDetails>> Update(int supplierId, List<AvailabilityRestriction> availabilityRestrictions)
     {
+        var propertyCode = availabilityRestrictions.First().PropertyCode;
```
[the remark this answers is not in the document]
Comment (AlexeyShok) [reply HappyTravel.Komoro.Api/Services/Availabilities/AvailabilityRestrictionService.cs:66]: Added comment

File: HappyTravel.Komoro.Api/Services/Statics/RoomTypeService.cs (hunk 1 above)
Comment (AlexeyShok) [reply HappyTravel.Komoro.Api/Services/Statics/RoomTypeService.cs:35]: Renamed

File: HappyTravel.Komoro.Api/Services/Statics/PropertyService.cs [hunk 4]
```diff
@@ -62,6 +62,10 @@ public async Task<int> GetId(int supplierId, string propertyCode)
 
         return property?.Id ?? 0;
     }
+    
+
+    public async Task<bool> IfExist(int supplierId, string propertyCode)
```
[the remark this answers is not in the document]
Comment (AlexeyShok) [reply HappyTravel.Komoro.Api/Services/Statics/PropertyService.cs:67]: Renamed

File: HappyTravel.Komoro.TravelClickChannelManager/Infrastructure/ErrorHelper.cs (hunk 2 above)
Comment (AlexeyShok) [reply HappyTravel.Komoro.TravelClickChannelManager/Infrastructure/ErrorHelper.cs:9]: Suggest another option. I'll fix it in a separate PR as contracts need to be changed

**Pull request merged**